src/test/java/uk/gov/hmcts/darts/common/service/impl/EodHelperMocks.java

```diff
@@ -37,6 +42,7 @@
 /**
  * Helper test class that mocks {@link EodHelper} entities and methods so that they can be used in unit tests.
  */
+@SuppressWarnings("PMD.NcssCount")
```

Comment:
Should be actioned instead of supressed

Reply:
This is on a test class so its not important

Reply:
It is still important for test classes as it improves test maintainability, readability and quality.

Reply:
ITs been removed